apps/next/pages/account.tsx

```diff
@@ -20,6 +20,7 @@ import Link from 'next/link';
 import { getSession, signOut } from 'next-auth/react';
 import axios from 'axios';
 import useSWR from 'swr';
+import { APIPaymentObject, PaymentPlan } from '@app/util/database/paymentHelpers';
 
 const Page: NextPage = () => {
```

Comment:
This page looks like ass. It's totally ok for now but before we release this lets at least spend 10 minutes making it look better.

![image](https://user-images.githubusercontent.com/12476526/144181012-7a725324-f9ce-42f1-9a9e-157334901fd3.png)